controller/src/settings/config.rs

```diff
@@ -84,7 +84,7 @@ pub struct AppSettings {
     pub esp_settings_enabled: BTreeMap<String, bool>,
 
     #[serde(default = "bool_true")]
-    pub bomb_timer: bool,
+    pub bomb_enabled: bool,
```

Comment:
We still need this as this is the bomb timer, which should not be forced being on the bomb itself. I personally still want to have the option to have a separate timer somewhere on screen.